content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/ledger-extensions-api-reference.md

```diff
@@ -364,7 +364,7 @@ The Ownable API provides the component to design ownable states and contracts; t
 
 An ownable state can be designed by implementing the `OwnableState` interface:
 
-``` kotlin
+``` java
```

Comment:
All the changes you made below this point should stay as `kotlin`, I can switch them over to `java` anyway.

So we can revert this part, or leave it, I don't mind.